.pipelines/build-and-push-images-tagged.yml

```diff
@@ -0,0 +1,57 @@
+# Azure DevOps Pipeline building rp images and pushing to int acr
+trigger: none
+pr: none
+
+variables:
+- template: vars.yml
+- name: tag
+
+jobs:
+- job: Build_and_push_images
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/v2')
```

Comment:
What is `v2`? Is it to  make sure that the tag starts with `v2`? Like `v20220119.00`?

Do we care in context of this pipeline if the tag starts with something else?